### ISSUE TITLE
Wave Audit

### ISSUE THREAD

**Issue by dcmorse** [contradb/contra#370]
# Wave Audit
Now that we have new wave figures, it's time to ge to work updating our dances. 

## Ocean Waves
- [x] Allison Jonjak [A 60th Birthday Dance (for Chris Deephouse)](http://contradb.com/dances/225/edit) 	
- [x] Allison Jonjak [A Proper Trifle](http://contradb.com/dances/399/edit) 	
- [x] Allison Jonjak [Apples and Concord Grapes](http://contradb.com/dances/102/edit) 	
- [x] Allison Jonjak [Dr. Whiting's Delight](http://contradb.com/dances/328/edit) 	
- [x] Allison Jonjak [Expert Bits](http://contradb.com/dances/118/edit) 	
- [x] Allison Jonjak [Holiday in the Wood](http://contradb.com/dances/313/edit) 	
- [x] Allison Jonjak [Hull's Victory](http://contradb.com/dances/433/edit) 	
- [x] Allison Jonjak [Passing Through](http://contradb.com/dances/478/edit) 	
- [x] Allison Jonjak [Pecan Pie](http://contradb.com/dances/430/edit) 	
- [x] Allison Jonjak [Proper Swingfest II](http://contradb.com/dances/432/edit) 	
- [x] Allison Jonjak [Reel Me Across the Ocean](http://contradb.com/dances/215/edit) 	
- [x] Allison Jonjak [Snake Oil Reel](http://contradb.com/dances/312/edit) 	
- [x] Allison Jonjak [Super Robin](http://contradb.com/dances/353/edit) 	
- [x] Allison Jonjak [Swingshot](http://contradb.com/dances/371/edit) 	
- [x] Allison Jonjak [The Big Bicep](http://contradb.com/dances/92/edit) 	
- [x] Allison Jonjak [Wave Goodbye to Winter](http://contradb.com/dances/444/edit) 	
- [x] Allison Jonjak [Weave Me the Sunshine](http://contradb.com/dances/226/edit) 	
- [x] Allison Jonjak [West By Midwest](http://contradb.com/dances/471/edit) 	
- [x] Chet Gray [Hotpoint Special](http://contradb.com/dances/143/edit) 	*private*
- [x] Chet Gray [Mange Tak](http://contradb.com/dances/141/edit) 	*private*
- [x] Chet Gray [Reel to Reel](http://contradb.com/dances/147/edit) 	*private*
- [x] Dane Wallinga [The Fabulous Couple](http://contradb.com/dances/475/edit) 	
- [x] Dave Morse [Across the Ohio](http://contradb.com/dances/69/edit) 	
- [x] Dave Morse [March of the Coffee Zombies](http://contradb.com/dances/89/edit) 	
- [x] Dave Morse [Summer of '84](http://contradb.com/dances/75/edit) 	
- [ ] Devin J. Pohly [Mountains in Sunny Windows](http://contradb.com/dances/172/edit) 	
- [x] Isaac Banner [One Fish, Two Fish, Right and Left Through Fish](http://contradb.com/dances/416/edit) 	
- [x] Isaac Banner [Reel'em In](http://contradb.com/dances/464/edit) 	
- [x] Isaac Banner [Two Waves, Two Ways](http://contradb.com/dances/388/edit) 	
- [x] Nicola Scott [West by Midwest](http://contradb.com/dances/285/edit) 	*private*
- [x] Robin Banerji [You Can Get There From Here](http://contradb.com/dances/445/edit) 	
- [ ] Terry Bachmann [Shadrack's Delight](http://contradb.com/dances/237/edit)

## Long Wave / Long Waves

- [x] to long waves (neighbor 3 in right hand, ladles face out, gentlespoons face in at partner's back) http://contradb.com/dances/150
- [x] Ladles walk in to center for long wave | balance | Gentlespoons walk in to long waves  while ladles walk in | balance http://contradb.com/dances/312
- [ ] Ladles walk forward to form a long wavy line and balance. | Ladles back out as gents walk in to form a wave and balance. http://contradb.com/dances/286
- [x] ladles walk forward to wavy line in middle | balance | ladles stay in place while gentlespoons walk forward to a wavy line in the middle | balance (there are interlinked wavy lines) http://contradb.com/dances/250
- [ ] ladles to the center to a long wavy line, balance, and stay there | gentlespoons to the center to a long wavy line, balance, and stay there , gentlespoons let go left, ladles let go right http://contradb.com/dances/456
- [x] gents allemande left ¾ to long wavy line of gents | balance | gents back out and ladies walk in to a long wavy line of ladies | balance http://contradb.com/dances/326
- [x] [Chuck the Budgie](http://contradb.com/dances/81)
- [x] [United we Dance](http://contradb.com/dances/78)
- [ ] [Going our Opposite Waves](http://contradb.com/dances/420) https://github.com/contradb/contra/issues/398 
- [X] [Rant and Roar](http://contradb.com/dances/270)
- [X] [Gallery Piece Transfer](http://contradb.com/dances/447)
- [X] [Jefferson and Liberty](http://contradb.com/dances/27)

**Comment by cranhandler** [comment]
This holds all the 'long waves' and 'a long wave' that got entered as custom. I fear we may have many, too, where a note got put (form a long wave) or (to a long wave) or (to waves where ladles face out). 

Could you do a database pull for wave and waves? Even if we get double hits, I don't want to miss any. If it's possible to just search the preamble and the in-line notes, that would likely catch all of them without lots of double-positives.

**Comment by dcmorse** [comment]
Don't worry, this was a comprehensive grepping at the time of writing. Notice how it got United We Dance despite "wave" only appearing in it's preamble.

**Comment by chetgray** [comment]
- [x] [Hotpoint Special](http://contradb.com/dances/143)
- [x] [Mange Tak](http://contradb.com/dances/141)
- [x] [Reel to Reel](http://contradb.com/dances/147) (and published)
- [x] [Four Have Sprung](http://contradb.com/dances/150) (though the 1st/2nd/3rd neighbors dialect rendering is buggy)

**Comment by cranhandler** [comment]
Thanks Chet, I noted Four Have Sprung in #309

**Comment by cranhandler** [comment]
In the long wave of http://contradb.com/dances/313/ , does the 0 beat 'form long wave' belong in the A2 where it fell, or in the B2 with its balance? 

![image](https://user-images.githubusercontent.com/16180173/40207954-a9dcafa2-59fc-11e8-8474-d6048aa48a11.png)

**Comment by cranhandler** [comment]
Common occurrence, making separate issue.

**Comment by cranhandler** [comment]
I want assistance with

 Robin Banerji You Can Get There From Here
 Allison Jonjak Apples and Concord Grapes
  Allison Jonjak Dr. Whiting's Delight
 Allison Jonjak Proper Swingfest II
 Allison Jonjak Swingshot
Dane Wallinga The Fabulous Couple

Texted owner of Mountains in Sunny Windows asking to fill out a moderation preference.
Emailed owner of West by Midwest of edit made
Emailed owner of You Can Get There From Here of edit made
Emailed owner of  One Fish, Two Fish, Right and Left Through Fish,  Reel'em In, Two Waves, Two Ways of new feature to implement

Left untouched: Dave Morse, Shadrach's Delight (unsure how to contact)

**Comment by cranhandler** [comment]
Squirrely dances: 
petronella to new wave http://contradb.com/dances/27
walk from old wave to new wave 20 Below
Cherokee Shuffle

**Comment by cranhandler** [comment]
I still need to pair on: 

Allison Jonjak Dr. Whiting's Delight
Allison Jonjak Proper Swingfest II
Allison Jonjak Swingshot
Dane Wallinga The Fabulous Couple

**Comment by cranhandler** [comment]
More wave thoughts from users: 
It looks pretty good, my $0.02 -

The default handed-ness of the move is still set to right-handed, but in the default case for the move the central hand should always be the left. It's the unusual case that a pass the ocean has a right-handed pivot.

The inclusion of the balance checkbox is nice, since it often gets bundled into the move, but because you also include the explanation of the move, the balance just gets hidden in the move description. Even excluding that I usually just break out a 'short waves balance' onto its own line on my dance cards. I personally will just stick to balance->everyone for my dances that need it (eg http://contradb.com/dances/388)

That said, also on the balance side of things, having a 'gents/larks' and a 'ladies/ravens' set of options under 'balance' would be excellent.

Other than that, it's much clearer now and I don't have to bend over backwards to try to include it, which is wonderful.

**Comment by cranhandler** [comment]
-[ ] http://contradb.com/dances/416

**Comment by cranhandler** [comment]
'take' right or left should be 'by' right or left

**Comment by cranhandler** [comment]
All outstanding dances are linked to their respective issues in https://github.com/contradb/contra/projects/5#card-10261051

**Comment by cranhandler** [comment]
https://contradb.com/dances/526/

**Comment by dcmorse** [comment]
I think we're just waiting on this one dance.
